From 26c0780746aecedf9d6fb73f0171620790638000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Hundeb=C3=B8ll?= <mnhu@prevas.dk>
Date: Wed, 22 Jul 2015 14:26:03 +0200
Subject: [PATCH] remove char arguments to boost::mutex constructors

---
 src/mongo/scripting/engine_v8-3.25.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mongo/scripting/engine_v8-3.25.cpp b/src/mongo/scripting/engine_v8-3.25.cpp
index b7f241c..14b2a43 100644
--- a/src/mongo/scripting/engine_v8-3.25.cpp
+++ b/src/mongo/scripting/engine_v8-3.25.cpp
@@ -382,7 +382,7 @@ void gcCallback(v8::GCType type, v8::GCCallbackFlags flags) {
 }
 
 V8ScriptEngine::V8ScriptEngine()
-    : _globalInterruptLock("GlobalV8InterruptLock"), _opToScopeMap(), _deadlineMonitor() {}
+    : _globalInterruptLock(), _opToScopeMap(), _deadlineMonitor() {}
 
 V8ScriptEngine::~V8ScriptEngine() {}
 
@@ -525,7 +525,7 @@ V8Scope::V8Scope(V8ScriptEngine* engine)
     : _engine(engine),
       _connectState(NOT),
       _cpuProfiler(),
-      _interruptLock("ScopeInterruptLock"),
+      _interruptLock(),
       _inNativeExecution(true),
       _pendingKill(false),
       _opId(0),
-- 
2.4.6

