From 6500ffdc89fb18355dd49c4e188562c067b96aee Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Hundeb=C3=B8ll?= <mnhu@prevas.dk>
Date: Wed, 22 Jul 2015 17:22:01 +0200
Subject: [PATCH] initialize arm_version variable in gyp

---
 common.gypi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common.gypi b/common.gypi
index 42ab572..ab5497b 100644
--- a/common.gypi
+++ b/common.gypi
@@ -11,6 +11,7 @@
     'gcc_version%': 'unknown',
     'clang%': 0,
     'python%': 'python',
+    'arm_version%' : '1',
 
     # Enable V8's post-mortem debugging only on unix flavors.
     'conditions': [
-- 
2.4.6

